spec/features/user_signs_in_spec.rb

```diff
@@ -8,12 +8,12 @@
   scenario 'specify valid credentials' do
     user = FactoryGirl.create(:user)
 
-    visit new_user_session_path
+    visit meetups_path
 
     fill_in 'Email', with: user.email
     fill_in 'Password', with: user.password
 
-    click_button 'Log in'
+    click_button 'Sign in'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.